feature/outing/src/commonMain/kotlin/team/aliens/dms/kmp/feature/outing/OutingApplicationScreen.kt

```
@@ -0,0 +1,34 @@
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import team.aliens.dms.kmp.core.designsystem.appbar.DmsTopAppBar
+import team.aliens.dms.kmp.core.designsystem.foundation.DmsTheme
+
+@Composable
+internal fun OutingApplication(
+    modifier: Modifier = Modifier
+) {
+
+}
```

detekt (1.23.7)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)